org.eclipse.xtext/src/org/eclipse/xtext/resource/persistence/PortableURIs.java

```diff
@@ -37,7 +37,7 @@
 import com.google.inject.Inject;
 
 /**
- * Portable URIs are based on names and therefore are independent of the concrete file pathes and fuile names the of
+ * Portable URIs are based on names and therefore are independent of the concrete file paths and file names of the
```

Comment:
independent _from_
"independent from" => separate from each ther
"independent of" => not influenced by each other

Reply:
@kthoms i believe we mean the latter here

Reply:
When in doubt, rephrase ;) `are based on names and do not depend on file paths ...`